src/components/scenes/Fio/FioAddressRegisterScene.tsx

```diff
@@ -201,7 +201,7 @@ export class FioAddressRegister extends React.Component<Props, State> {
         try {
           await checkIsDomainPublic(fioPlugin, domain)
         } catch (e: any) {
-          showError(e.message)
+          showWarning(e.message, { trackError: false })
```

Comment:
`String(error)` is safer

Reply:
Also, why aren't we logging this warning?

Reply:
Per our convo. this was going to be changed to remove the setTimeout and make checkFioAddress async. However the setTimeout is there to debounce the action. Fixing this makes this a much larger change.

Reply:
I'm less concerned about the setTimeout and more just about cleaning up the showWarning to instead get the error string from the return of `checkIsDomainPublic` to set it to the error state on the component and remove the try..catch

Reply:
understood. done